public/scripts/report.js

```diff
@@ -1,6 +1,6 @@
 (function() {
   // add listener to get location button
-  var locationButton = document.getElementById('getLocationButton');
+  var locationButton = document.getElementById('location');
```

Comment:
ahh yea I covered this in your PR as well - I think if we add a hidden input with the id `location` then we can just get the value from that when we submit the form - then yea these can be changed back to `getLocation` and `searchLocation` respectively